_docs_use-cases/use-case-consumption-get-shared-versions-of-a-repositoryattribute.md

```diff
@@ -64,7 +64,7 @@ This use case allows you to retrieve a list of own shared [IdentityAttributes]({
 
 - No LocalAttributes can be returned if the `attributeId` correlates to an unknown LocalAttribute.
 - No LocalAttributes can be returned if the `attributeId` correlates to a RelationshipAttribute.
-- No LocalAttributes can be returned if the `attributeId` correlates to an IdentityAttribute with a `shareInfo`.
+- No LocalAttributes can be returned if the `attributeId` correlates to an own shared or peer shared IdentityAttribute.
```

Comment:
...an own shared or **a** peer shared IdentityAttribute.

Reply:
I think you can use both and I like it better the way I formulated it.